test/test_network_constructors.jl

```diff
@@ -118,7 +118,11 @@ end
     c_sys14_dc = build_system("c_sys14_dc")
     systems = [c_sys5, c_sys14, c_sys14_dc]
     objfuncs = [GAEVF, GQEVF, GQEVF]
-    constraint_names = [:RateLimit_ub__Line, :RateLimit_lb__Line, :Bus_active_power_balance]
+    constraint_names = [
+        :RateLimit_ub__Line,
+        :RateLimit_lb__Line,
+        PSI.constraint_name(PSI.NODAL_BALANCE_ACTIVE, PSY.Bus),
```

Comment:
This test seems that it needs refinement. Why is this the only constraint name tested this way?

Reply:
I'm not sure, I didn't write these these probably Clayton did.